src/Helper/HeadTitle.php

```diff
@@ -151,15 +151,11 @@ public function getDefaultAttachOrder()
     private function getTitleItemCallback()
     {
         if (! $this->isTranslatorEnabled() || ! $this->hasTranslator()) {
-            return function ($item) {
-                return $item;
-            };
+            return fn($item) => $item;
```

Comment:
Most of these should be `static fn`?

Reply:
implemented https://github.com/laminas/laminas-view/pull/111/commits/a97613597d3ee45bf2162a4bf804e089bd70ba6e

Reply:
Oh, wow, one of them is problematic:

```

1) LaminasTest\View\Helper\FlashMessengerTest::testCanDisplayListOfMessages
Cannot bind an instance to a static closure
```

Reply:
should be fixed now https://github.com/laminas/laminas-view/pull/111/commits/7a5e318201da6f360f722c551da0363b80b76447